assets/src/stories-editor/blocks/amp-story-page/test/__snapshots__/index.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`amp/amp-story-page block edit matches snapshot 1`] = `
 <div
-  style="background-position:cover;background-repeat:no-repeat;background-size:cover"
+  style="background-position:;background-repeat:no-repeat;background-size:cover"
```

Comment:
Hmm, that is unrelated. Should be fixed by c3908503f6401c4d26fef1f02cbf167632f9b73b.

Reply:
Humm, I'm getting this without it:
![Screen Shot 2019-05-10 at 6 18 36 PM](https://user-images.githubusercontent.com/3294597/57541767-0fbaf780-7350-11e9-94e0-5dcc10e2bfe8.png)

Reply:
Yes some tests are expected to fail at the moment, as outlined in the original PR. The failed snapshot should be fixed now though as I said. Might need to merge the latest changes into this branch

Reply:
Yep, I was referring to Snapshots fail only.

I've merged the latest changes and ran it again, now the snapshots test passes but still with an update:
https://github.com/ampproject/amp-wp/pull/2278/files#diff-3b752b7a9ffae3b6b7cef57de4b9709dL5

Reply:
That looks OK. Thanks for double checking.